functions/src/server.ts

```diff
@@ -10,7 +10,7 @@ import * as openapi from '../build/swagger.json';
 
 export const app = express();
 
-const port = process.env.PORT || 3001;
+const port = process.env.PORT || 2999;
```

Comment:
Change this back to `3001` instead of `2999`.

Reply:
@chsunwoo1002 ^^

Reply:
@aomi fixed it!